.github/workflows/generator.yml

```diff
@@ -0,0 +1,50 @@
+name: "Generator"
+
+on:
+  merge_group:
+  pull_request:
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
+  push:
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
+  workflow_call:
+    inputs:
+      ref:
+        description: "The git ref to check"
+        type: string
+        required: true
```

Comment:
This is only necessary if we're calling this from a different workflow (e.g. in the release workflow). Since we're not doing that, I suggest removing this section.
```suggestion
```